### ISSUE TITLE
[tests] Firefox 119 fails ARIA tests in Karma

### ISSUE THREAD

**Issue by nolanlawson** [salesforce/lwc#3753]
Due to Firefox 119 [shipping](https://bugzilla.mozilla.org/show_bug.cgi?id=1785412) ARIA reflection but [deviating](https://bugzilla.mozilla.org/show_bug.cgi?id=1853209) from Chrome's/Safari's behavior, our Karma tests break in Firefox Nightly.

<details><summary>Failed tests</summary>

```
FAILED polyfills aria-properties ariaSort should return null if the value is not set 
FAILED polyfills aria-properties ariaSort should remove the attribute if the property is set to null 
FAILED polyfills aria-properties ariaMultiLine should remove the attribute if the property is set to null 
FAILED polyfills aria-properties ariaMultiLine should return null if the value is not set 
FAILED polyfills aria-properties ariaLevel should remove the attribute if the property is set to null 
FAILED polyfills aria-properties ariaLevel should return null if the value is not set 
FAILED polyfills aria-properties ariaBusy should return null if the value is not set 
FAILED polyfills aria-properties ariaBusy should remove the attribute if the property is set to null 
FAILED polyfills aria-properties ariaRelevant should return null if the value is not set 
FAILED polyfills aria-properties ariaRelevant should remove the attribute if the property is set to null 
FAILED polyfills aria-properties ariaValueMax should return null if the value is not set 
FAILED polyfills aria-properties ariaValueMax should remove the attribute if the property is set to null 
FAILED polyfills aria-properties ariaAtomic should return null if the value is not set 
FAILED polyfills aria-properties ariaAtomic should remove the attribute if the property is set to null 
FAILED polyfills aria-properties ariaRowSpan should remove the attribute if the property is set to null 
FAILED polyfills aria-properties ariaRowSpan should return null if the value is not set 
FAILED polyfills aria-properties ariaColSpan should remove the attribute if the property is set to null 
FAILED polyfills aria-properties ariaColSpan should return null if the value is not set 
FAILED polyfills aria-properties role should return null if the value is not set 
FAILED polyfills aria-properties role should remove the attribute if the property is set to null 
FAILED polyfills aria-properties ariaRowCount should return null if the value is not set 
FAILED polyfills aria-properties ariaRowCount should remove the attribute if the property is set to null 
FAILED polyfills aria-properties ariaDisabled should return null if the value is not set 
FAILED polyfills aria-properties ariaDisabled should remove the attribute if the property is set to null 
FAILED polyfills aria-properties ariaRowIndex should remove the attribute if the property is set to null 
FAILED polyfills aria-properties ariaRowIndex should return null if the value is not set 
FAILED polyfills aria-properties ariaColIndex should return null if the value is not set 
FAILED polyfills aria-properties ariaColIndex should remove the attribute if the property is set to null 
FAILED polyfills aria-properties ariaOrientation should remove the attribute if the property is set to null 
FAILED polyfills aria-properties ariaOrientation should return null if the value is not set 
FAILED polyfills aria-properties ariaColCount should remove the attribute if the property is set to null 
FAILED polyfills aria-properties ariaColCount should return null if the value is not set 
FAILED polyfills aria-properties ariaValueNow should return null if the value is not set 
FAILED polyfills aria-properties ariaValueNow should remove the attribute if the property is set to null 
FAILED polyfills aria-properties ariaInvalid should remove the attribute if the property is set to null 
FAILED polyfills aria-properties ariaInvalid should return null if the value is not set 
FAILED polyfills aria-properties ariaKeyShortcuts should return null if the value is not set 
FAILED polyfills aria-properties ariaKeyShortcuts should remove the attribute if the property is set to null 
FAILED polyfills aria-properties ariaRoleDescription should return null if the value is not set 
FAILED polyfills aria-properties ariaRoleDescription should remove the attribute if the property is set to null 
FAILED polyfills aria-properties ariaExpanded should return null if the value is not set 
FAILED polyfills aria-properties ariaExpanded should remove the attribute if the property is set to null 
FAILED polyfills aria-properties ariaValueMin should return null if the value is not set 
FAILED polyfills aria-properties ariaValueMin should remove the attribute if the property is set to null 
FAILED polyfills aria-properties ariaPosInSet should remove the attribute if the property is set to null 
FAILED polyfills aria-properties ariaPosInSet should return null if the value is not set 
FAILED polyfills aria-properties ariaHasPopup should return null if the value is not set 
FAILED polyfills aria-properties ariaHasPopup should remove the attribute if the property is set to null 
FAILED polyfills aria-properties ariaReadOnly should remove the attribute if the property is set to null 
FAILED polyfills aria-properties ariaReadOnly should return null if the value is not set 
FAILED polyfills aria-properties ariaAutoComplete should remove the attribute if the property is set to null 
FAILED polyfills aria-properties ariaAutoComplete should return null if the value is not set 
FAILED polyfills aria-properties ariaCurrent should return null if the value is not set 
FAILED polyfills aria-properties ariaCurrent should remove the attribute if the property is set to null 
FAILED polyfills aria-properties ariaSetSize should remove the attribute if the property is set to null 
FAILED polyfills aria-properties ariaSetSize should return null if the value is not set 
FAILED polyfills aria-properties ariaLabel should return null if the value is not set 
FAILED polyfills aria-properties ariaLabel should remove the attribute if the property is set to null 
FAILED polyfills aria-properties ariaPressed should return null if the value is not set 
FAILED polyfills aria-properties ariaPressed should remove the attribute if the property is set to null 
FAILED polyfills aria-properties ariaModal should return null if the value is not set 
FAILED polyfills aria-properties ariaModal should remove the attribute if the property is set to null 
FAILED polyfills aria-properties ariaHidden should return null if the value is not set 
FAILED polyfills aria-properties ariaHidden should remove the attribute if the property is set to null 
FAILED polyfills aria-properties ariaSelected should return null if the value is not set 
FAILED polyfills aria-properties ariaSelected should remove the attribute if the property is set to null 
FAILED polyfills aria-properties ariaLive should return null if the value is not set 
FAILED polyfills aria-properties ariaLive should remove the attribute if the property is set to null 
FAILED polyfills aria-properties ariaRequired should return null if the value is not set 
FAILED polyfills aria-properties ariaRequired should remove the attribute if the property is set to null 
FAILED polyfills aria-properties ariaChecked should return null if the value is not set 
FAILED polyfills aria-properties ariaChecked should remove the attribute if the property is set to null 
FAILED polyfills aria-properties ariaPlaceholder should remove the attribute if the property is set to null 
FAILED polyfills aria-properties ariaPlaceholder should return null if the value is not set 
FAILED polyfills aria-properties ariaMultiSelectable should return null if the value is not set 
FAILED polyfills aria-properties ariaMultiSelectable should remove the attribute if the property is set to null 
FAILED polyfills aria-properties ariaValueText should remove the attribute if the property is set to null 
FAILED polyfills aria-properties ariaValueText should return null if the value is not set
FAILED component html-properties global html properties role getter
```

</details>

We should probably fix this in advance so that, when Firefox 119 ships, our tests don't suddenly start breaking in CI.

**Comment by nolanlawson** [comment]
This may also end up being a nothingburger if Firefox fixes [the bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1853209) before shipping.

**Comment by nolanlawson** [comment]
Removed "up for grabs" because it's debatable whether we want to fix this since Firefox themselves might.

**Comment by nolanlawson** [comment]
Firefox fixed the bug, plus #3792 would have fixed it anyway
